WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewModel.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import Combine
+import OSLog
+
+struct CardReader {
```

Comment:
I was going to comment "Have you tried to consume Yosemite.CardReader?", which made me realize that we might have lost that model object when I merged develop into our feature branch yesterday. I'll submit a PR to bring it back soon